boltops-tools/kubes
# Merger options

This is a 🙋‍♂️ feature or enhancement.
This is a 🧐 documentation change.


## Summary

Allow user to change core merging behavior. 

.kubes/config.rb

```ruby
Kubes.configure do |config|
  # ...
  config.merger.options = {overwrite_arrays: true} # default
end
```

## Context

Some users prefer the Array be added to during an array. IE: base/deployment.yaml sidecar containers.

## How to Test

Configure the option and run `kubes apply` and confirm that merged behavior is what you want.

## Version Changes

Patch
